Server/src/services/poi.service.ts

```diff
@@ -97,11 +104,17 @@ export default class POIService{
      */
     public static async getPOITrackDistanceKm(poi: POI): Promise<number | null>{
         // get closest track if none is given
-        const poiPos: GeoJSON.Feature<GeoJSON.Point> = JSON.parse(JSON.stringify(poi.position))
-        if (poiPos == null || poiPos.properties == null || poiPos.properties["trackKm"] == null) {
+        const poiPos = GeoJSONUtils.parseGeoJSONFeaturePoint(JSON.parse(JSON.stringify(poi.position)))
```

Comment:
stringifying and immediately parsing an object seems unnecessary/waseful. I guess you only want an object with the correct type for you to work on it or is there a reason why you would not want to work on the `poi.position` directly?